packages/iov-ethereum/src/ethereumconnection.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { EthereumConnection } from "./ethereumconnection";
+import { TestConfig } from "./testconfig";
+
+describe("EthereumConnection", () => {
+  const base = TestConfig.base;
+  const nodeChainId = TestConfig.chainId;
+  const minHeight = TestConfig.minHeight;
+
+  it(`can be constructed for ${base}`, () => {
+    const connection = new EthereumConnection(base, nodeChainId);
```

Comment:
Those tests should be pending if `ETH_ENV` is not set. See bns tests for example code